conformance/Basic.java

```diff
@@ -20,13 +20,24 @@
 
 class Basic {
   @NonNull Object cannotConvertNullableToNonNull(@Nullable Object nullable) {
-    // test:cannot-convert:Object? to Object!
     // test:expression-type:Object?:nullable
+    // test:sink-type:Object!:cannotConvertNullableToNonNull#return
+    // test:cannot-convert:Object? to Object!
     return nullable;
   }
 
   @Nullable Object canConvertNonNullToNullable(@NonNull Object nonNull) {
     // test:expression-type:Object!:nonNull
+    // test:sink-type:Object?:canConvertNonNullToNullable#return
```

Comment:
We could skip the name of the function here, since it should be unambiguous.

```suggestion
    // test:sink-type:Object?:return
```

Reply:
Skipping SGTM.